.github/workflows/build.yaml

```diff
@@ -51,3 +51,19 @@ jobs:
           git push
           git tag -a "${tag_name}" -m "Release brew formula for Pace CLI ${tag_name}"
           git push --tags
+      - name: Update CLI docs
+        env:
+          GITHUB_TOKEN: ${{secrets.GH_TOKEN}}
+          BOT_USERNAME: ${{secrets.BOT_USERNAME}}
+          BOT_EMAIL: ${{secrets.BOT_EMAIL}}
+        run: |
+          make docs
+          git config --global user.email "${BOT_EMAIL}"
+          git config --global user.name "${BOT_USERNAME}"
+          git clone "https://git:${GITHUB_TOKEN}@github.com/getstrm/pace.git"
```

Comment:
Maybe an explicit branch checkout would be wise here (i.e. `alpha`).

Reply:
done